travis/cfg/travis_run_pylint_pr.cfg

```diff
@@ -31,24 +31,29 @@ disable=all
 #   duplicate-xml-record-id - W7902
 #   eval-used - W0123
 #   incoherent-interpreter-exec-perm - W8201
+#   invalid-commit - E8102
 #   javascript-lint - W7903
 #   manifest-author-string - E8101
 #   manifest-deprecated-key - C8103
 #   manifest-required-author - C8101
 #   manifest-required-key - C8102
 #   manifest-version-format - C8106
+#   method-compute - C8108
+#   method-inverse - C8110
 #   method-required-super - W8106
```

Comment:
I don't understand
Do you wish remove the line `#   method-required-super - W8106`?
This line don't have changes for this PR
Could you help me to understand?